### ISSUE TITLE
Is there a way to use existing text noised images to train noise model(aim to remove watermark)?

### ISSUE THREAD

**Issue by qazwsdc** [yu4u/noise2noise#36]
I want to remove image or video watermark.
In your method, we must split noise model before train a model.
But it's difficult to split text(watermark) noise model because the watermark is already on images.
So is there a way to use existing watermarked images to train a model?

Thanks!

**Comment by yu4u** [comment]
You can insert synthetic watermarks into non-watermarked images to create training images.

**Comment by qazwsdc** [comment]
Yes I know.But it's difficult to split watermark because the watermark is already on images and have many changes.It's not a simple color watermark.
So is there a way to directly use watermarked images to train?

**Comment by yu4u** [comment]
> So is there a way to directly use watermarked images to train?

Basically, no.
The original paper assumed that we could get multiple images generated by adding different watermarks to the original image, but it hardly happens.

**Comment by ghost** [comment]
hey，Hey, brother, the same needs, have you found a solution?

**Comment by CabbageWust** [comment]
我也在干这个事，正在训练， 不知道能不能达到效果

**Comment by ghost** [comment]
@CabbageWust https://www.zhihu.com/question/333086833/answer/841650555

**Comment by ghost** [comment]
@CabbageWust 我试过了，效果一般般，不能完全去干净，应该是那里需要修改

https://www.v2ex.com/t/599596

**Comment by CabbageWust** [comment]
多谢， 稍微看了下， 应该和我做法一样（把水印随机贴在同一图像的两个位置，作为一个样本对），但是目前训练的模型测试结果完全不像样..... 你能说下你训练的 样本数量、输入尺寸、 epoch、 和最后的loss吗  0.0
